lang/string_extractor/parsers/generic.py

```diff
@@ -6,8 +6,8 @@
 def parse_generic(json, origin):
     name = ""
     comment = []
-    if "//" in json:
-        comment.append(json["//"])
+    if "//~" in json:
+        comment.append(json["//~"])
```

Comment:
If none of the comments extracted by this is useful to translators then you could simply remove these two lines. Any further comments for translators can be directly added to the names and/or descriptions by add a `//~` member to them.

Reply:
I'd like to keep this. It is not written anywhere exactly where comments can be. Comments here at the top level will apply to all nested strings, which is quite intuitive and would be strange if it didn't work at some point.

Reply:
> It is not written anywhere exactly where comments can be.

It's written in the exact documentation you refered to above:

https://github.com/CleverRaven/Cataclysm-DDA/blob/fc58a18b361fe074ff039aa391ee60ab9b4fa0ef/doc/JSON_INFO.md?plain=1#L461-L469

> Comments here at the top level will apply to all nested strings, which is quite intuitive and would be strange if it didn't work at some point.

I suppose it can be, although I'd much prefer the syntax to be consistent across all objects, and AFAIK using `//~` in the top level is only supported in a few object types.

Reply:
> AFAIK using // as comments for translators in the top level is only supported in a few object types and you've only changed one of them to //~ too.

ouch, my bad, i checked it rn, generic_parser doesnt work as i thought. i removed these lines, thank you for the help.